createjs-lib/index.d.ts

```diff
@@ -70,6 +70,7 @@ declare namespace createjs {
         addEventListener(type: string, listener: (eventObj: Object) => void, useCapture?: boolean): Function;
         addEventListener(type: string, listener: { handleEvent: (eventObj: Object) => boolean; }, useCapture?: boolean): Object;
         addEventListener(type: string, listener: { handleEvent: (eventObj: Object) => void; }, useCapture?: boolean): Object;
+        addEventListener(type: string, listener: Function, useCapture?: boolean): Function;
```

Comment:
`(...args: any[]) => any` is stricter than `Function`, for example, since it disallows `new Function("...")`